apps/vr-tests-react-components/src/stories/ProgressBar.stories.tsx

```diff
@@ -1,8 +1,15 @@
 import * as React from 'react';
 import { storiesOf } from '@storybook/react';
 import { ProgressBar } from '@fluentui/react-progress';
+import { PauseAnimationDecorator, TestWrapperDecoratorFixedWidth } from '../utilities/TestWrapperDecorator';
+import { StoryWright, Steps } from 'storywright';
 
 storiesOf('ProgressBar converged', module)
+  .addDecorator(PauseAnimationDecorator)
+  .addDecorator(TestWrapperDecoratorFixedWidth)
+  .addDecorator(story => (
+    <StoryWright steps={new Steps().snapshot('default', { cropTo: '.testWrapper' }).end()}>{story()}</StoryWright>
+  ))
```

Comment:
This doesn't appear to be correctly cropping the screenshots. Could you look into fixing it to crop the screenshots to just the content? It may have to do with the order of these decorators?